drivers/storage/ide/uniata/id_probe.cpp

```diff
@@ -2006,10 +2017,24 @@ UniataClaimLegacyPCIIDE(
         goto del_do;
     }
 
+#ifdef __REACTOS__
+    ExFreePool(resourceList);
+    ExFreePool(oldResList);
+#endif
+
     KdPrint2((PRINT_PREFIX "ok %#x\n", status));
     BMList[i].ChanInitOk |= 0x80;
 
     return status;
+
+#if __REACTOS__
```

Comment:
why #ifdef at the other locations but only #if here?

Reply:
```suggestion
#ifdef __REACTOS__
```

Reply:
Done 👍